src/server/usage/classes/collector_set.js

```diff
@@ -73,12 +72,12 @@ export class CollectorSet {
    * Call a bunch of fetch methods and then do them in bulk
    * @param {Array} collectors - an array of collectors, default to all registered collectors
    */
-  bulkFetch(callCluster, collectors = this._collectors) {
-    if (!Array.isArray(collectors)) {
+  bulkFetch(callCluster, collectors = this) {
```

Comment:
Should we rename `collectors` to `collectorSet` to avoid any potential confusion? By `collectors`, I'd assume we're doing with an array of collectors versus a typed `CollectorSet` object